### DISCUSSION TITLE
Low light / night usage

### DISCUSSION THREAD

**Discussion by benboughton1** [geezacoleman/OpenWeedLocator#112] [General]
What’s the current thought on potential to use at night? Given you use a regular rgb camera I’m guessing a normal led light would work as an active light source? We use our weedit a bit at night and also thinking any automated applications would probably prefer a night option.

**Comment by geezacoleman** [comment]
We tested it in the original publication at nighttime with a Stedi 40W light and it performed ok - it's just very important to tune the threshold settings to the particular light you end up using. I imagine speed would be more limited too due to more image blur, but perhaps on an autonomous rover it isn't so much of an issue.

For the later versions, a [Stedi 90W LED](https://www.stedi.com.au/90w-led-flood-light.html) light worked better, the first version was quite dark on the edges of the field of view so missed a lot of weeds there.

You can see them mounted under each camera unit here:
![image](https://github.com/geezacoleman/OpenWeedLocator/assets/51358498/27b95cb4-c9a6-401c-9919-f243aae56140)
